gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+              <NxTable.Cell>NxList.Link supports any HTML attribute that's normally supported
+                by <NxCode>{'<a>'}</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>NxList.Text</NxTable.Cell>
+              <NxTable.Cell>truncate</NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>Whether the list item text should be truncated. Displays ellipses
+                to cut off text.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>NxList.Subtext</NxTable.Cell>
+              <NxTable.Cell>truncate</NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>Whether the list item text should be truncated. Displays ellipses
+                to cut off subtext.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>NxList.Error</NxTable.Cell>
+              <NxTable.Cell>errorMessage</NxTable.Cell>
+              <NxTable.Cell>string</NxTable.Cell>
+              <NxTable.Cell>yes</NxTable.Cell>
+              <NxTable.Cell>Error message to display.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>onClick</NxTable.Cell>
+              <NxTable.Cell>function</NxTable.Cell>
+              <NxTable.Cell>yes</NxTable.Cell>
+              <NxTable.Cell>Function for retry.</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </section>
+
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__title">
+          <h3 className="nx-h3">Component Breakdown</h3>
+        </header>
+        <NxP><NxCode>NxList</NxCode> comprises of several children components.
+          A reference of each component and where they must be placed is as follows:
+        </NxP>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Component</NxTable.Cell>
+              <NxTable.Cell>Location</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>Parent; Top-Level</NxTable.Cell>
+              <NxTable.Cell>This is the parent list class. By default, it has no bullets. In order to have a
+                bulleted list, the prop <NxCode>bulleted</NxCode> can be provided.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Title</NxCode></NxTable.Cell>
```

Comment:
The title isn't a part or responsibility of the list itself so I don't think we should have this component

Reply:
> The title isn't a part or responsibility of the list itself so I don't think we should have this component

`NxList.Title` has been removed from `NxList`. Users will now have to provide their own `NxH3` as their `NxList` title.
12bca56fd2c81312c66eef2f7dfa14ba86b42e95